gtests/net/mptcp/regressions/close_mpj_timeout.pkt

```diff
@@ -0,0 +1,36 @@
+// connection initiated by packetdrill
+--tolerance_usecs=100000
+`../common/defaults.sh
+sysctl -q net.mptcp.close_timeout=1`
+
+// TODO keep using ADD_ADDR to keep test passing, until DSS values in packets 9 ~ 15 is understood
```

Comment:
(I'm not sure what you are referring to: is it linked to a Packetdrill issue?

Reply:
this is a left-over from the original script (used to take "inspiration"). I'll drop the comment in the next revision